src/components/BaseLayerToggler/BaseLayerToggler.js

```diff
@@ -129,7 +150,19 @@ class BaseLayerToggler extends Component {
   }
 
   componentWillUnmount() {
+    const { layerService } = this.props;
     unByKey([this.postRenderKey, this.moveEndKey]);
+    layerService.un('change:visible', e => this.updateState(e));
```

Comment:
you unlisten a different function , it should be the same that you use when you listen.

Reply:
But I listen/unlisten the same:
`('change:visible', e => this.updateState(e));`
Doesn't it work so?

Reply:
you create 2 differents objects . How the js engine is suppose to know the are the same function.